src/test/java/com/uid2/shared/store/RotatingSiteStoreTest.java

```diff
@@ -58,8 +58,10 @@ private Site addSite(JsonArray content, int siteId, String name, boolean enabled
         JsonObject site = new JsonObject();
```

Comment:
Same comment here about `ObjectMapper`

Reply:
I'm missing some understanding - how can we use `ObjectMapper` here? I think we can avoid using this `JsonObject()` and the numerous `site.put`s below - is this still using `OBJECT_MAPPER.readValue`?

Reply:
how's this?
```
private Site addSite(JsonArray content, int siteId, String name, String description, boolean enabled, boolean visible, long created, String... domains) throws IOException {

    Site s = new Site(siteId, name, description, enabled, new HashSet<>(), new HashSet<>(List.of(domains)), visible, created);

    String jsonString = OBJECT_MAPPER.writeValueAsString(s);
    JsonNode jsonNode = OBJECT_MAPPER.readTree(jsonString);

    content.add(jsonNode);

    return s;
}
```

Reply:
Actually you might be able to use mapper.valueToTree() or mapper.convertValue(object, JsonNode.class)